mne/bem.py

```diff
@@ -318,6 +318,9 @@ def make_bem_solution(surfs, verbose=None):
         logger.info('Homogeneous model surface loaded.')
     else:
         raise RuntimeError('Only 1- or 3-layer BEM computations supported')
+    if bem['surfs'][0]['np'] > 10000:
+        warn('The bem surface has %s data points. 5120 should be enough.'
```

Comment:
Should probably warn that it might not save properly, and to DRY it this can be a `_check_bem_size` private function